travel_platform/travel_app/views.py

```diff
@@ -1,12 +1,45 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
-
-from django.http import Http404
-
+from django.core.cache import cache
 from django.shortcuts import render
 
-from .models import Query
+from api.izi_travel.izi_travel_data import *
 
 
 def index(request):
     return render(request, 'travel_app/main.html')
+
+
+def get_city_tourist_info(request):
+    city = request.GET['desired_location']
+    info = cache.get(city)
```

Comment:
How do you feel working with caching? Is it difficult to understand?
Or it's clear enough?